mapstruct-on-gradle-testcp-with-lombok/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id "io.freefair.lombok" version "6.0.0-m2"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+
+    annotationProcessor 'org.mapstruct:mapstruct-processor:1.4.2.Final'
+    compileOnly 'org.mapstruct:mapstruct-processor:1.4.2.Final'
+
+    testAnnotationProcessor 'org.mapstruct:mapstruct-processor:1.4.2.Final'
+    testCompileOnly 'org.mapstruct:mapstruct-processor:1.4.2.Final'
```

Comment:
Same comment here. Why do we need mapStruct on the `compile` class path?

Reply:
same as above